components/user_settings/display/manage_timezones/index.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {connect} from 'react-redux';
+import {ActionCreatorsMapObject, bindActionCreators, Dispatch} from 'redux';
+import {updateMe} from 'mattermost-redux/actions/users';
+import {ActionFunc, ActionResult} from 'mattermost-redux/types/actions';
+import {UserProfile} from 'mattermost-redux/types/users';
+
+import ManageTimezones from './manage_timezones';
+
+type Actions ={
```

Comment:
This is total nitpicking over styling, but would you mind fixing the whitespace here? I'm not sure why ESLint didn't pick this up since it usually complains about stuff like this.
```suggestion
type Actions = {
```

Reply:
resolved